models/articles-models.js

```diff
@@ -41,7 +41,6 @@ exports.fetchArticles = () => {
   ORDER BY created_at DESC
   ;`;
   return db.query(queryStr).then(({ rows }) => {
-    console.log(rows);
     return rows;
   });
 };
```

Comment:
I know this is minor - but I would recommend doing as little as possible per commit - mainly as this commit message is "changing file names" but more changes are actually taking place
Something to bear in mind 😀